video2_elements_and_syntax/README.md

```diff
@@ -0,0 +1,17 @@
+## class 2
```

Comment:
Mention popular properties(their usage) of these tags are more helpful in the future. eg: img tag: src, alt